src/adapters/RoomsAdapter.js

```diff
@@ -28,6 +28,11 @@ export default class RoomsAdapter extends WebexAdapter {
    * @property {RoomType}  type    The type of the room. @see RoomType enum
    */
 
+  /**
+   * @typedef {Object} ActivityData
+   * @param {string} date The date to render a time ruler for
```

Comment:
I would like to specify the format for the string
```suggestion
   * @param {string} date The date to render a time ruler for. Must be a valid date-time string
   *     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#Date_Time_String_Format
```